Localization/LanguageInstance.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Localization
+{
+    public static class LanguageInstance
```

Comment:
I don't see any reason to separate this instance into another class. Instead I suggest to make `TranslationTable` class a singleton, you can do this easy with an `Instance` property and null-coalescing operators.

Reply:
I don't agree on this as `TranslationTable` is designed to be corresponding to an .ini file, instead of the current language table of the running program. For instance, the program may load all the .ini file in something like `/Resources/Translation/*.ini` and make all the `TranslationTable.LanguageName` into a list to let user select the language, in the future.

Reply:
There's no reason for this to be a separate class *at this stage of development*. It's not hard to change from singleton class to non-singleton when that actually gets implemented, which may never happen because of complexity of reloading everything.

If you don't want to make it a singleton the way I described, then it's okay, but we still have to get rid of this redundant class because it just creates unnecessary separation which makes the code harder to navigate.

Reply:
Alright. Now I removed this class and moved the field into `TranslationTable.Instance`. But I did not choose to use `??=` operator to prevent possible (although I've not found) `L10N()` invokes before the Instance get assigned.